torch/nn/utils/parametrize.py

```diff
@@ -389,6 +394,9 @@ def right_inverse(self, X: Tensor) -> Union[Tensor, Sequence[Tensor]]
         tensor_name (str): name of the parameter or buffer on which to register
             the parametrization
         parametrization (nn.Module): the parametrization to register
+    Keyword args:
+        unsafe (bool): a boolean flag that denotes whether the parametrization breaks the invariants. Default: `False`
```

Comment:
Perhaps it's better to be more explicit, as we do not refer to these as "invariants" in the documentation?
> whether the parametrization may change the dtype and shape of the tensor.